Jenkinsfile

```diff
@@ -0,0 +1,51 @@
+pipeline {
```

Comment:
Instead of this, I'd try to use the [`packagePipeline`](https://github.com/ocf/shared-pipeline/blob/master/vars/packagePipeline.groovy) if you can to avoid duplication. It should be something like this, although the `Makefile` targets would need to be changed to just have `package_stretch` as the only target.

Then this whole file would just be this, although it would need some changes to the `Makefile` to achieve this:

```
packagePipeline(
    dists: ['stretch'],
)

// vim: ft=groovy
```

(I should go update marathon-deb too, I think it can be simplified a fair amount)